### PR TITLE
feat(aio): docs landing page and styles

### PR DESCRIPTION
**Pull request by sjtrimble**
**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")
```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Other... Please describe:
```

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[x] No
```

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: aio/src/styles/1-layouts/_top-menu.scss [hunk 1]
```diff
@@ -16,6 +16,11 @@
   height: 36px;
 }
 
+.home-logo-link {
+  height: 40px;
+  margin-right: 20px;
```
Comment (gkalpak) [inline aio/src/styles/1-layouts/_top-menu.scss]: I think some left margin would look nice on mobile. But maybe it's something to be fixed for `.nav-link`s on small screens.

File: aio/content/marketing/docs.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        <p class="card-footer">View Quickstart</p>
+
+    </a>
+
+    <a href="../api" class="card">
+        <h2>API Reference</h2>
+        <p>An advanced reference of all Angular Classes, Methods, etc.</p>
+        <p class="card-footer">View Quickstart</p>
+    </a>
+<div>
```
Comment (gkalpak) [inline aio/content/marketing/docs.html]: I assume this is a temporary, throw-away file, right?

Comment (sjtrimble) [inline aio/content/marketing/docs.html]: No, what do you mean?

**Review by wardbell — APPROVED** [review]
LGTM

File: aio/content/marketing/docs.html (hunk 2 above)
Comment (gkalpak) [inline aio/content/marketing/docs.html]: Hm...

1. The links seem wrong (e.g. the `..` are not necessary and the `Developer Guide` card links to the non-existent page).

2. And hard-coding `v4.0.0` is also unexpected (for me at least :smiley:).

3. And finally this sentence seems totally non-production ready :grin:
> If you're looking for Angular vs2 documents you'll find them at <a href="#">v2.angular.io</a>

**Pull request merged**